src/instruments/src/EFB/Efb.tsx

```diff
@@ -119,6 +118,9 @@ const Efb = () => {
 
     useEffect(() => {
         document.documentElement.classList.add(`theme-${theme}`, 'animationsEnabled');
+        if (!AircraftVersionChecker.isVersionChecked) {
```

Comment:
Now that I think of it, we wanna delay this until the user is definitely loaded in the cockpit - some people have crashes when the popup shows before.

I think the sentry popup has some logic to do this.

Reply:
Ok - will look into this

Reply:
Added wait for A32NX_IS_READY which basically is the same.
Also moved the check to the index.tsx and our of the EFB.tsx - think it fits better there